### PR TITLE
Completed lesson02 Exercises

### PR DESCRIPTION
**Pull request by vinhle000**
Completed Exercises for gridprinter.py, fizzbuzz.py, and series.py

### CODE REVIEW

**Comment by vinhle000** [comment]
Added list, slice, and stringformat exercises for Lesson03

File: students/vinh_le/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            fruit_list.remove(fruit_list[list_count])
+            fruit_count -= 1
+        else:
+            print('Please answer with a "yes" or "no". ')
+
+
+    print(fruit_list)
+
+
+def series4(fruit_list):
```
Comment (majstorpoki) [inline students/vinh_le/lesson03/list_lab.py:86]: You did not test this one.

File: students/vinh_le/lesson03/slice_lab.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+
+def exchange_first_last(seq):
+    new_sequence = (seq[-1:len(seq)] + seq[1:-1] + seq[0:1])
+    return new_sequence
+
+def remove_every_other(seq):
+    new_sequence = seq[0:-1:2]
```
Comment (majstorpoki) [inline students/vinh_le/lesson03/slice_lab.py:7]: 0 not needed

File: students/vinh_le/lesson03/slice_lab.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+def reverse_elements(seq):
+    return seq[::-1]
+
+
+def last_first_middle_third(seq):
+    third_count = int(len(seq)/3)
+
+    first = seq[0: third_count]
+    middle = seq[third_count: (2 * third_count)]
```
Comment (majstorpoki) [inline students/vinh_le/lesson03/slice_lab.py:24]: Good

**Review by majstorpoki — APPROVED** [review]
Thanks. GREAT.

**Pull request merged**